src/components/Image/Image.test.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
+
+import ImageWET from '@components/Image';
+
+describe('Image Tests', () => {
+  test('Test image component', () => {
```

Comment:
group all the tests for shapes under one "describe". we should have a chat about this to organize the tests a bit better.

Also note: Each test should only be testing one thing. For example if you want to test the 'alt' prop then it should have its own test. The shape 'rounded' should have its own test. we can talk about this as well :)

Reply:
I will fix the tests and their grouping